db/migrate/20170511120047_remove_container_group_failed_sync_event.rb

```diff
@@ -0,0 +1,14 @@
+class RemoveContainerGroupFailedSyncEvent < ActiveRecord::Migration[5.0]
+  class Relationship < ActiveRecord::Base; end
+
+  class MiqEventDefinition < ActiveRecord::Base; end
```

Comment:
@Fryguy Are these class definitions enough or should they still include `self.inheritance_column = :_type_disabled`

There seem to be a number of more recent migrations that define classes with the `_type_disabled`.